Gemfile

```diff
@@ -3,12 +3,13 @@ source "https://rubygems.org"
 gem "byebug"
 gem "capybara"
 gem "capybara-screenshot"
+gem "capybara-select2"
```

Comment:
Might be good to separate out the `Gemfile` changes into a separate commit.

Reply:
Good point, might also be better to have each of the tests in separate commits